.github/workflows/publish-to-npm.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish packages to NPM
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release-to-npm:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 18
+          registry-url: 'https://registry.npmjs.org'
+
+      - run: npm ci
+
+      - run: npm run compile
+
+      - name: Publish to npm
+        # only publish if a release has been created
+        if: ${{ steps.release.outputs.releases_created }}
```

Comment:
Probably I'm missing something but where this output comes from?

Reply:
good catch, I had copied this from the workflow that we use in contrib, which had other steps in addition to this.

Reply:
removed in [0fc05ef](https://github.com/open-telemetry/opentelemetry-js/pull/5067/commits/0fc05eff53f618f99d06222f25fec4bbf8cb7cae)